configs/mask_rcnn/mask_rcnn_r50_caffe_fpn_detectron2-poly_1x_coco.py

```diff
@@ -16,7 +16,7 @@
         type='Resize',
         img_scale=[(1333, 640), (1333, 672), (1333, 704), (1333, 736),
                    (1333, 768), (1333, 800)],
```

Comment:
We have a standard naming convention for config files in v2.0. Please refer to the [doc](https://github.com/open-mmlab/mmdetection/blob/v2.0/docs/CONFIG.md#config-name-style).

Reply:
I think the naming of mask_rcnn_r50_caffe_fpn_detectron2-poly_1x_coco.py is an issue at the original v2.0 branch. Shall I correct it here?

Reply:
Oh it should have been solved in #2446, you may just leave this file unchanged to avoid conflicts.